package.json

```diff
@@ -7,7 +7,7 @@
     "gen-cuba-front": "bin/gen-cuba-front.js"
   },
   "engines": {
-    "node": ">=8.0"
+    "node": ">=10.17"
```

Comment:
This introduces breaking change in theory, so we will need to bump major version